meetings/2022-03-01.md

```diff
@@ -0,0 +1,134 @@
+
+## Minutes
+
+### sai-java
+
+* JB: I completed functional parts and now just add coverage. I can walk you all through next week and I welcome feedback.
+
+### Fix AccessNeedDescription pointing back to AccessNeedGroup 
+
+[PR#239](https://github.com/solid/data-interoperability-panel/pull/239)
+
+* EP: Made one small fix; seems good to go.
+
+### Terminology: Access Consent 
+
+[Issue#242](https://github.com/solid/data-interoperability-panel/issues/242)
+
+* JB: If you haven't filled out your preferences please take a minute to do it now. https://pollunit.com/en/polls/osk8gjzizehvo-vqkbobda
+* JB: Last week the question was if there is free site providing such pools. This one seems to be working well and I was able to set up this pool using free tier.
+* JB: In short Consent is bit loaded term and it may be better to use different name.
+* ePa: Most people gave at least one dot for * Authorization so it seems no objections.
+* JB: With authorization it seems nice since it includes Authorization term.
+* EP: I want to make sure we don't use this term elsewhere
+* JB: In the spec we have section talking about Authorization broadly, we could consider renaming it to avoid any confusions.
+* EP: We don't have any structures with that name
+* EP: Should we consider Data Access Authorization? It seems different to authorize Data Access than authorize Data.
+* WT: I agree we don't need to add access but now we seem to have bit redundant terminology - Access Authorization.
+* LD: I would stick with Data Authorization, I don't see need to add anything in between.
+* JB: What about the main Access Authorization? Is it strange to say it?
+* LD: Maybe we should make it consistent.
+* EP: Authorizing data to me sounds like authorizing incorporation of data rather than sharing of data.
+* JB: Data Authorization must have an Access Authorization it is part of, maybe that provides the context.
+* ePa: I see it as good improvement. We may need to do one more, hopefully final, iteration when we have some examples going beyond Data Registrations. So Generic authorizations for data browser like apps and admin level / trusted authorizations.
+* JB: Section 7 of the spec is called Data Authorization. It broadly describes authorization flows. Does anyone have any suggestions for new good name.
+* LD: Why not *Authorization Flows* as you just referred to it.
+* JB: I would also consider Authorization Patterns and *Authorization Scenarios*
+
+ACTION: JB: to make PR with renaming and change section 7 title.
+
+### Source of truth for access control
+
+* WT: Cf. discussion LDB & WT: Are these the consents/grants in the registries, or the decisions of the AS? In particular: can a (UMA) AS overrule AA consents/grants, either temporary or by changing the ACPs/ACRs?
+* WT: Laurens and I discussed AA AS and their interaction. What is the source of truth for the access control. If Access Grants set by AA it would be weird if AS couldn't overwrite it.
```

Comment:
```suggestion
* WT: Laurens and I discussed AA AS and their interaction. What is the source of truth for the access control? If Access Grants set by AA, it would be weird if AS couldn't overwrite it.
```